### DISCUSSION TITLE
Welcome to LightZone Discussions!

### DISCUSSION THREAD

**Discussion by ktgw0316** [Aries85/LightZone#277] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cjwag1** [comment]
Good day.  I have been playing with LightZone since around 2013.  Just for my own enjoyment.  I was using it with my canon 600D.  Then went to a 800D and had problems that were quickly remedied.  A Magenta haze was covering the raw files.  Now I have the R6 Mark III and the .CR3 files are doing the same.  Was able to download and install the LightZone-5.0.0beta3-20250525-windows-64bit.msi file but still the problem. 
<img width="884" height="738" alt="image" src="https://github.com/user-attachments/assets/4a43d400-3785-4601-95c4-aac9fdb98f3c" />

 I noticed in reading that is mentioned support of the R6M2 .CR3 files.  Did Canon change something again?  Any Idea if or when this new format may become available?

Thanks for all your hard work.

**Reply by sfink16** [reply]
Welcome @cjwag1 
Lightzone uses DCRaw for older cameras.  Support for those cameras ended 5+ years ago.   Libraw for newer cameras is the newer raw conversion tool that Lightzone uses.  Click [HERE](https://www.libraw.org/supported-cameras) to see the list of cameras supported in Libraw.  The latest Canon cameras are the R5 MKII and the R8, I believe.  I don't know when the next Libraw release is scheduled for but hopefully the R6 MKIII will be included.  None of the other free raw convertors (Darktable, Rawtherapee, etc.) that I tried work for that camera either.

Obviously Canon's DPP4 can be used to convert their R6 MKIII camera and probably Adobe's paid Lightroom program can convert the R6 MKIII images.  I use DPP4 and convert the images to 16-bit Tiff images.  Tiff images can be opened in Lightzone but those images are quite large, slowing down Lightzone when working with them.

**Reply by cjwagnersr** [reply]
Looking at RawDigger, and having downloaded their 'Free Trial'  It seems to open the CR3 file without problems.  I do not know what RawDigger does or what it is actually used for, beyond my knowledge, but they say it uses Libraw to open newer camera formats. As their homepage shows...
<img width="1014" height="336" alt="image" src="https://github.com/user-attachments/assets/6857b2c0-8681-4260-8d22-58d235466051" />
Having tinkered with software years ago, I understand there are many variables when changing just one line of code.  Maybe something in LightZone that doesn't play nice.  Maybe Rawdigger has modified their Librawf.dll file to work with their software.  

I appreciate all you do with this fantastic software.  I'm just going down a rabbit hole, trying to read and understand more about how this stuff all works.  

Thanks again for a great product.

**Reply by sfink16** [reply]
@cjwag1  

I saw that too.  I didn't go as far as you did, but isn't Digger simply an analyzing tool, NOT a raw convertor.  I don't know how that translates to Libraw since they haven't had a new release in a while, but we may need to consider patience until the next release comes out.

**Reply by sfink16** [reply]
@cjwag1 @cjwagnersr 

While waiting for the next version of LibRaw, there are two options to consider.

1. Shotwell use the embedded JPG images of the R6 MKIII CR3 files. While
2. [Online Convertor CR3 to DNG ](https://www.onlineconverter.com/cr3-to-dng) works as well, albeit one image at a time.
